MAINTAINERS.yml

```diff
@@ -896,8 +896,8 @@ Devicetree:
     - galak
     - rruuaanng
   files-regex:
-    - dts/bindings/.*zephyr.*
-    - dts/bindings/[^,]+$
+    - ^dts/bindings/.*zephyr.*
```

Comment:
we have `files-exclude`

Reply:
Yes, but the goal of this line is to only include the top level dts/bindings folder, not sure what would be the advantage of going through the effort to figure out every single other pattern to exclude and keep updating the excludes whenever a new one is discovered

Reply:
sorry, I meant we have `files-regex-exclude`

Reply:
using the exclude variant is just better for readability, i understand it will end up doing the same.

Reply:
I really think that would not be more readable than just the ^ character denoting the start of the path